src/wallet/spend.cpp

```diff
@@ -198,7 +193,7 @@ CoinsResult AvailableCoins(const CWallet& wallet,
             // Filter by spendable outputs only
             if (!spendable && only_spendable) continue;
 
-            int input_bytes = GetTxSpendSize(wallet, wtx, i, (coinControl && coinControl->fAllowWatchOnly));
+            int input_bytes = CalculateMaximumSignedInputSize(output, &wallet, false);
```

Comment:
why not `use_max_sig=coinControl && coinControl->fAllowWatchOnly` ?

Reply:
I'd like to remove the flag